analyticsclient/tests/test_course.py

```diff
@@ -149,6 +149,23 @@ def test_problems(self):
         self.assertEqual(body, self.course.problems())
 
     @httpretty.activate
+    def test_typology(self):
+
+        body = [
+            {
+                "chapter_id": "week2",
+                "video_type": 2,
+                "problem_type": 1,
```

Comment:
I guess I was thinking that by leaving them out here, it would be a better test and we'd know when the constants were changed. But they're not really being used anywhere in this repo itself so that's not actually the case. I will change it :)